sdk/src/ssi/credential.go

```diff
@@ -0,0 +1,44 @@
+package ssi
+
+import (
+	"github.com/TBD54566975/ssi-sdk/credential"
+	"github.com/TBD54566975/ssi-sdk/credential/signing"
+	"github.com/TBD54566975/ssi-sdk/crypto"
+	"github.com/goccy/go-json"
+)
+
+func SignVerifiableCredentialJWT(keyID, keyType string, privateKey, vcJSONBytes []byte) []byte {
```

Comment:
Do we need to add godoc so that mobile devs know what this does without having to understand this code?

Reply:
will add